### ISSUE TITLE
It seems AltTab interfers with the latest IntellIJ EAP and sometimes "shadows" keyboard shortcuts

### ISSUE THREAD

**Issue by mfn** [lwouis/alt-tab-macos#652]
**Describe the bug**

This will not be easy to describe 😄 

Basically, I started to having issues with PhpStorm in that sometimes I would perform keyboard interaction and suddenly PhpStorm would not accept any keypress anymore.

I was already about to open a bug at https://youtrack.jetbrains.com/ _but_ during writing the report there I did more analysis and suddenly I found out I don't have this problem when I quit AltTab 😢 

To me, it's 100% reproducible with  "PhpStorm 2020.3 EAP Build #PS-203.4449.24, built on October 8, 2020" and AltTab 6.7.3 (I just updated it to verify)

What the gif below shows:
- I press my custom 2nd keystroke command <kbd>^g b</kbd> to get the list of branches
- I select one for rename, but press escape
- **no keyboard shortcuts work anymore**, not even the basic PhpStorm ones
- When I press <kbd>^g</kbd> before invoking the 2nd keystroke, a list of all available 2nd stroke options appear at the very bottom
  ![image](https://user-images.githubusercontent.com/87493/95962343-8cd08300-0e06-11eb-9350-f10087fd5bfd.png)
- in case after I canceled the branch rename, only one 2nd keystroke option appears at the bottom; instead of all
  ![image](https://user-images.githubusercontent.com/87493/95962439-a8d42480-0e06-11eb-9901-5fd14f64c00f.png)
- Once I quit AltTab, it works normally
- If I did not have AltTab running in the first place, I do not experience this problem at all

Basically it looks like something is "over shadowing" the PhpStorm window. Note that for this to occur, I **did not** even invoke the AltTab feature. It was just started/running without ever being invoked.

🤷‍♀️

![alttab_phpstorm_interferece](https://user-images.githubusercontent.com/87493/95961882-f603c680-0e05-11eb-9888-618956e74492.gif)

PS: I'm happy to also report this on YouTrack, but after by pure gut feeling I found the relation to AltTab, I decided to open it here first, thanks

**Comment by mfn** [comment]
Ok, seems there's more to properly investigate here. I further found out:
- it does not matter whether this operation is invoked via keyboard shortcut or not
- I can do the same with just the mouse to get to the "branch rename" dialog and it triggers the same issue

I currently don't know any other way to trigger this, so so far it seems that the "branch rename" modal dialog from PhpStorm is somehow "special" here and interferes with AltTab in some way.

**Comment by mfn** [comment]
I could reproduce this on the latest IntelliJ IDEA EAP with no customizations whatsoever:
- downloaded from https://www.jetbrains.com/idea/download/download-thanks.html?type=eap,rc&platform=mac
- started up and open a random Java / git project
- navigated to the branch rename dialog, closed it
- no further keyboard shortcuts where accepted

**Comment by lwouis** [comment]
Thank you very much for this Sherlock Holmes investigation @mfn! I had a blast reading the intricate explanations 😄 

I would try to reproduce locally and see what could be at play here. Based on the symptoms, I have no clue what it could be.

Quick question in the meanwhile: did this happen / does it happen with [AltTab 6.7.2](https://github.com/lwouis/alt-tab-macos/releases/tag/v6.7.2)?

**Comment by mfn** [comment]
> does it happen with AltTab 6.7.2?

Yes!

**Comment by lwouis** [comment]
I was able to reproduce locally! At first I couldn't, on my IntelliJ, but then I installed the EAP, and there it reproduces. This seems to be exclusive to the EAP version then.

**Comment by lwouis** [comment]
I found the root cause!

It has nothing to do with keyboard events, contrary to what I assumed. It's the [same exotic issue I witnessed in the past with another app called Protege](https://github.com/lwouis/alt-tab-macos/issues/314#issuecomment-626334872). Somehow, just subscribing to the accessibility events of IntelliJ EAP breaks the scenario with branch renaming. Subscribing to accessibility events (using `AXObserverCreate` + `AXObserverAddNotification`) should never affect an app. It's supposed to be passive observation for accessibility app like a screenreader, or AltTab. However somehow it affects the app in this case, and in Protege's case.

It looks like a clear bug from macOS API here. Observing an app shouldn't break it. Also it's only happening for `kAXFocusedUIElementChangedNotification`, not the dozen other notification types.
